### ISSUE TITLE
Investigate strange batch download behaviour

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/pvoutput#28]
1. On 28th Aug, why did it freeze up?  Waiting too long for network to timeout?

2. On Sat 31 Aug, why didn't it skip to ~50% before downloading?

3. What's going on here:

```
2019-09-01 12:18:28,949 - pvoutput - INFO - system_id 9570: Got empty timeseries back for 2012-03-31
2019-09-01 12:19:16,906 - pvoutput - INFO - system_id 9570: 143543 total rows downloaded
```

**Comment by JackKelly** [comment]
3 is fine - that's expected behaviour.  Still need to look into 1 & 2

**Comment by JackKelly** [comment]
2 is fixed in #31 

1 is tracked in #32
